src/components/NcRichText/NcRichText.vue

```diff
@@ -217,15 +223,15 @@ export default {
 					},
 					prefix: false,
 				})
-				.processSync(this.useMarkdown
-				// In order to correctly show newlines in Markdown,
-				// each newline contains a non-breaking space
-					? this.text.slice()
-						.replace(/\n>\n/g, '\n>\u00A0\n')
-						.replace(/\n{2,}/g, (match) => {
-							return '\n' + '\n\u00A0\n'.repeat(match.length - 1)
-						})
-					: this.text)
+				.processSync(this.text.slice()
+					// escape special symbol "<" to not treat text as HTML
+					.replace(/</gmi, '&lt;')
+					// unescape special symbol ">" to parse blockquotes
+					.replace(/&gt;/gmi, '>')
+					// add a non-breaking space to each newline to show them
+					.replace(/\n>\n/g, '\n>\u00A0\n')
+					.replace(/\n{2,}/g, (match) => '\n' + '\n\u00A0\n'.repeat(match.length - 1)),
```

Comment:
It works and it's not a part of this PR. But non-breaking space is still a space, it's additional character in the rendered text, it's being selected, copied... :(

Reply:
Copy from `NcRichText` output
![image](https://github.com/nextcloud-libraries/nextcloud-vue/assets/25978914/5ce2cb0b-21a0-4c74-860d-ddd37d81efd7)

Reply:
We could then proceed with the same approach Github does it: add margin-bottom after some tags (p, ul, ol, blockquote), and get rid of new lines

Reply:
Line1
Line2


Line3

Reply:
It seems that GitHub just doesn't allow to add any number of new lines and works like MD here. `\n{2,}` works as a paragraph separator and just adds a paragraph with margin. Only single `\n` works as new line.

Reply:
But we can do it in a separate PR later if it is not a part of this PR, don't feel blocked :D

Reply:
GitHub `markdown-body` styles for the reference

https://github.githubassets.com/assets/primer-15214e5f9104.css

Reply:
Other usages of Markdown across NC apps:

https://github.com/nextcloud/spreed/blob/144f7a61de0689f72c27313538ea883ff28efe2b/src/components/MessagesList/MessagesGroup/Message/Message.vue#L1057
https://github.com/nextcloud/welcome/blame/ef1bdfe51aea6516ccbbd06b54e91439ee502510/src/views/Dashboard.vue#L117
https://github.com/nextcloud/integration_github/blob/585ea8e1fd5f93b44d13ddd8ac6d00080ececb0f/src/views/GithubIssuePrReferenceWidget.vue#L657

Additional spaces between `\n` should be removed, to restore initial parsing (before my first changes) and not break markdown in other apps unintentionally.

But regarding the styles, looks like everyone does it in theirs own way.
We could either stick with that, and don't define any additional styles in the vue-lib, or setup some common things, like `margin` or `color` alignments